frontend/src/pages/members/MemberDependantsTab.tsx

```
@@ -0,0 +1,189 @@
+          Dependants ({dependants.length})
+        </h2>
+        <button
+          className="px-4 py-2 rounded-lg transition-colors flex items-center gap-2"
+          style={{ backgroundColor: colors.action.primary, color: '#ffffff' }}
+          onMouseEnter={(e) => {
+            e.currentTarget.style.opacity = '0.9';
+          }}
+          onMouseLeave={(e) => {
+            e.currentTarget.style.opacity = '1';
+          }}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)